src/bsp/src/stm32/src/InterlocManager.cpp

```diff
@@ -1,7 +1,26 @@
 #include "InterlocManager.h"
 #include "bsp/BSPContainer.h"
 #include <Task.h>
+#define NB_CALIB_TRIES 10
 
+bool isFrameGood(UWBRxFrame frame, ILogger& logger) {
```

Comment:
isFrameOk, also you could make the function as a private method of the interlocmanager, would avoid passing the logger